zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2913,6 +2997,56 @@ components:
           type: string
           description: The name of the tenant.
 
+    TenantItem:
+      description: Tenant search response item.
+      type: object
+      properties:
+        key:
+          type: integer
+          description: The unique system-generated internal tenant ID.
+          format: int64
+        name:
+          type: string
+          description: The tenant name.
+        tenantId:
+          type: string
+          description: The unique external tenant ID.
+        assignedMemberKeys:
+          type: array
+          description: The set of keys of members assigned to the tenant.
+          items:
+            type: integer
+            format: int64
+
+    TenantSearchQueryRequest:
+      description: Tenant search request
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      properties:
+        filter:
+          allOf:
```

```suggestion
        filter:
          description: The tenant search filters.
          allOf:
```

🔧 Please add descriptions to all schema properties, for the docs.